i18n/nl/cosmic_edit.ftl

```diff
@@ -1,27 +1,67 @@
+cosmic-text-editor = COSMIC-tekstbewerker
 new-document = Nieuw document
 open-project = Open project
+todo = Te doen
 
 # Context Pages
 
+## About
+git-description = Git commit {$hash} op {$date}
+
 ## Document statistics
 document-statistics = Document statistieken
 word-count = Aantal woorden
 character-count = Tekens
 character-count-no-spaces = Tekens (zonder spaties)
-line-count = Lijnen
+line-count = Regels
+
+## Git management
+git-management = Git-beheer
+git-management-description = Git-beheer is een hulpmiddel voor ontwikkelaars dat wordt gebruikt voor versiebeheer.
+git-management-loading = Git-beheer wordt geladen...
+stage = Voorbereiden
+staged-changes = Voorbereide wijzigingen
+unstage = Niet voorbereiden
+unstaged-changes = Niet voorbereide wijzigingen
+
+## Project search
+project-search = Project zoeken
+
+## Prompt save changes
+prompt-save-changes-title = Wijzigingen niet opgeslagen
+prompt-unsaved-changes = Er zijn wijzigingen die niet zijn opgeslagen. Opslaan?
+cancel = Annuleren
+discard = Wijzigingen verwerpen
+save-all = Alles opslaan
 
 ## Settings
 settings = Instellingen
 
 ## Appearance
 appearance = Weergave
 theme = Thema
+match-desktop = Afstemmen op systeemkleuren
+dark = Donker
+light = Licht
+syntax-dark = Donkere syntaxmarkering
+syntax-light = Lichte syntaxmarkering
 default-font = Standaard lettertype
 default-font-size = Standaard lettertypegrootte
 
 ### Keyboard shortcuts
 keyboard-shortcuts = Toetsenbord snelkoppelingen
-enable-vim-bindings = Schakel Vim snelkoppelingen in
+enable-vim-bindings = Schakel Vim-snelkoppelingen in
+
+# Find
```

Comment:
Line 55 to 64 are based on the translation of the find-and-replace feature in Libreoffice